apps/go-mod-validator/action.yaml

```
@@ -0,0 +1,17 @@
+name: go-mod-validator
+description: "Validates dependencies in go.mod files are tested"
+
+inputs:
+  github-token:
+    description:
+      "Github token with read permissions on all smartcontractkit repos"
+    required: true
+    default: ${{ github.token }}
+  go-mod-dir:
+    description: "Common directory where go.mod files are located"
+    required: true
```

```suggestion
    default: ${{ github.token }}
  go-mod-dir:
    description: "Common directory where go.mod files are located"
```

I dont think we need to set required since we set a default anyway, unless I am mistaken

---

yes, make senes. Done